portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,14 @@
 #greeting-container {
   margin-top: 20px;
 }
+#aggies {
+    align-content: left;
+    width: 100px;
+    height: 100px;
```

Comment:
It's generally better practice to use css classes for stuff like this.  I would define 3 classes (feel free to rename):

1) .thumbnail - controls the size of all your images of this class
2) .align-left
3) .align-right

Also, for the left/right it might be better to use a grid styling in a container div if you plan to add more images

Reply:
Can you explain what you mean about the use of classes?

Reply:
Sure thing!  See https://www.w3schools.com/cssref/css_selectors.asp for a more detailed reference, but basically:

html elements are most commonly matched to css styles through either their:
1) id - each html element can only have a single id and it's supposed to be unique on the page.  id selectors aren't all that useful unless you're clearly dealing with some kind of root-level container that will always be unique.
2) type - this will match every html element of a given type.  This is useful if you want to specify page-wide styling defaults (e.g. all \<h1\> tags have font-size 20px)
3) classes - this is by far the most useful option.  each html element can specify any number of classes (e.g. \<img class="thumbnail left-align"\> would have class "thumbnail" and "left-align").  This allows you to reuse general styles across multiple elements, and also merge styles.